spec/controllers/articles_controller_spec.rb

```diff
@@ -1,22 +1,22 @@
 require 'spec_helper'
 
 RSpec.describe ArticlesController, type: :controller do
-  describe '#index' do
+  describe 'GET #index' do
     describe 'show the list to articles saved' do
```

Comment:
This is not a scope that you need to represent, this is an action, so it needs to be in a `it` block.
```
describe 'GET #index' do
  it 'lists all the articles' do
```

Don't upload blank specs.